Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -526,6 +526,10 @@
   #define HAS_MANUAL_MOVE_MENU 1
 #endif
 
+#if ANY(HAS_MARLINUI_MENU, ULTIPANEL_FEEDMULTIPLY, SOFT_RESET_ON_KILL)
```

Comment:
The `ULTIPANEL_FEEDMULTIPLY` and `SOFT_RESET_ON_KILL` options aren't available here. They are both defined in `Configuration_adv.h`.